include/wx/qt/cursor.h

```diff
@@ -28,8 +27,9 @@ class WXDLLIMPEXP_CORE wxCursor : public wxCursorBase
              int hotSpotX = 0, int hotSpotY = 0);
 #endif
 
-    QCursor m_qtCursor;
-
+    virtual wxPoint GetHotSpot() const;
+    QCursor &GetHandle() const;
```

Comment:
Wouldn't `GetQCursor()` be a better name? (At least after seeing the `GetQColor()` change above.)